entrypoint/restart/restart_suite_test.go

```diff
@@ -0,0 +1,45 @@
+package restart
```

Comment:
https://github.com/rancher/distros-test-framework/pull/50/files#r1414602620
lets rename the package to restartservice